submissions/cynthiaNwakaeme/CSS/style.css

```diff
@@ -0,0 +1,286 @@
+*,body{
+    font: 15px/1.5, Arial, Helvetica, sans-serif;
```

Comment:
@Cynthia-Berry I think you should use font size instead of this. Then put `Arial, Helvetica, sans-serif` into attribute `font-family`

Reply:
@ToluAkin You have been amazing.
thank you changes has been effected.